### PR TITLE
[TACHYON-1450] TACHYON_WORKER_MEMORY_SIZE does not work as expected when bare metal total memory is over 100GB

### PR DESCRIPTION
**Pull request by BrickXu**
`free -b` would truncate value if total memory is over 100GB, and TACHYON_WORKER_MEMORY_SIZE will not work as expected.

### CODE REVIEW

**Comment by calvinjia** [comment]
Jenkins, test this please.

File: bin/tachyon-mount.sh [hunk 1]
```diff
@@ -137,7 +137,7 @@ function mount_ramfs_linux() {
   fi
 
   mem_size_to_bytes
-  FREE_MEM=`free -b | grep "^Mem" | awk '{print $2}'`
+  FREE_MEM=$(($(cat /proc/meminfo | awk 'NR==1{print $2}') * 1024))
```
Comment (calvinjia) [inline bin/tachyon-mount.sh]: This is actually total memory and not free memory right? I think we always wanted total memory, perhaps change the variable name?

Comment (BrickXu) [inline bin/tachyon-mount.sh]: It should be `TOTAL_MEM` here, it make me confused at first time.

**Comment by BrickXu** [comment]
@calvinjia done.

**Comment by calvinjia** [comment]
LGTM

**Comment by javading** [comment]
Not all cases are like this!!!
memory size: 256g
(1) when  CentOS release 6.7 (Final)  free -b is error...
            [root@hadoop19 tachyon-0.7.1-hadoop2.6-build]# free -b
                           total       used       free     shared    buffers     cached
             Mem:    2708683776 1727548743 9811350323    1273856 2160799744 1144124252
             -/+ buffers/cache: 5618164940 2146867281 
             Swap:   1258291159          0 1258291159 
(2) when    CentOS release 6.6 (Final) or  	CentOS release 6.2 (Final) is true...
            [root@hdh010017139236 ~]# free -b
                          total       used       free     shared    buffers     cached
             Mem:    270851489792 159177363456 111674126336     270336 6998409216 92962271232
             -/+ buffers/cache: 59216683008 211634806784
             Swap:   107374178304          0 107374178304

**Comment by BrickXu** [comment]
@javading Thanks for these cases, very great!

**Pull request merged**